### PR TITLE
added backgroundColor property

### PR DESCRIPTION
**Pull request by lutianxiong**
1. added backgroundColor property. 
2. removed transparency property.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -117,16 +113,15 @@ class BarcodeMask extends React.Component {
     });
   };
 
-  _applyMaskFrameTransparency = () => {
-    let transparency = 0.6;
+  _applyMaskFrameBackgroundColor = () => {
```
Comment (shahnawaz) [inline index.js]: As this method is now providing more than one style `{ backgroundColor: backgroundColor, flex: 1 }`, let's rename it to `_applyMaskFrameStyle`?

File: index.js [hunk 2]
```diff
@@ -234,14 +229,14 @@ const defaultProps = {
   edgeHeight: 20,
   edgeColor: '#FFF',
   edgeBorderWidth: 4,
-  transparency: 0.6,
   showAnimatedLine: true,
   animatedLineColor: '#FFF',
   animatedLineHeight: 2,
   lineAnimationDuration: 1500,
+  backgroundColor: 'rbga(0, 0, 0, 0.6)'
```
Comment (shahnawaz) [inline index.js]: Could you also update declarations https://github.com/shahnawaz/react-native-barcode-mask/blob/master/index.d.ts#L10 ?

**Comment by shahnawaz** [comment]
@lutianxiong Please update README.md? https://github.com/shahnawaz/react-native-barcode-mask/blob/master/README.md#transparency

Much appreciated!

File: index.js (hunk 1 above)
Comment (shahnawaz) [reply index.js]: Could you also confirm if this default backgroundColor is actually being applied if no backgroundColor prop is passed from outside? `rgba(0, 0, 0, 0.6)`

Apparently, the project I am working with isn't showing the default bkg color, but it shows bkg color if I passed it like `<BarcodeMask backgroundColor='rgba(0,0,0,0.6)'/>`

With transparency it was working by providing default transparency.

Comment (lutianxiong) [reply index.js]: I have added the default backgroundColor

File: index.js (hunk 2 above)
Comment (lutianxiong) [reply index.js]: It's done.

**Pull request merged**